bound/kt/src/main/kotlin/web5/sdk/crypto/keys/Jwk.kt

```diff
@@ -0,0 +1,71 @@
+package web5.sdk.crypto.keys
+
+import web5.sdk.rust.JwkData as RustCoreJwkData
+
+/**
+ * Partial representation of a [JSON Web Key as per RFC7517](https://tools.ietf.org/html/rfc7517).
+ * Note that this is a subset of the spec.
+ */
+data class Jwk(
```

Comment:
since `RustCoreJwkData` is solely a data structure, without any instance methods (or another way of saying this is, since `Jwk` in the APID 0.1.0 is solely a data structure without instance or static methods or constructors)... I wonder if you could just re-export `import web5.sdk.rust.JwkData as RustCoreJwkData` instead of rewriting the data class? 🤔 trying to exercise my kt muscles... if yes, then the same would hold true for any type in the APID which is solely a data structure, such as `Document` or `VerificationMethod` or `Service` or `DocumentMetadata` or `InputDescriptor` and so on...

if so, it would save a lot of overhead, right? because conversions would be unnecessary

Reply:
this may be unwise, so just an idea I'm throwing out for consideration

Reply:
so but yeah to be clear, if we have a `CLASS` in the APID which has anything other than a set of `PUBLIC DATA` then we'll definitely have to wrap the rust core bound code b/c UDL interfaces don't conform to the APID